src/test/scala/cromwell/engine/db/slick/SlickTestDatabase.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.engine.db.slick
+
+import java.sql.Connection
+
+import cromwell.util.DatabaseConfig
+import liquibase.Liquibase
+import liquibase.database.DatabaseConnection
+import liquibase.resource.{FileSystemResourceAccessor, ResourceAccessor}
+
+// Modified from https://tillias.wordpress.com/2012/11/10/unit-testing-and-integration-testing-using-junit-liquibase-hsqldb-hibernate-maven-and-spring-framework/
+trait SlickTestDatabase {
+  SlickTestDatabase.checkInitialized()
```

Comment:
It feels kind of convoluted that `checkInitialized` is a noop whose invocation side-effects the loading of the object whose constructor invokes `start`; perhaps there's a more straightforward way to accomplish this?